docs/code/best_practices.md

```diff
@@ -27,3 +27,22 @@ TODO: Add more notes about how to isolate subsystems from component specifics
 *  Use `requestAnimationFrame`
 *  Avoid constant synchronous DOM reads/writes
 *  Reduce the number of composite layers
+
+### Typescript
```

Comment:
Do we want to mentioned the definite assignment operator and commenting where it is assigned?